schema/manifest_test.go

```diff
@@ -89,6 +89,23 @@ func TestManifest(t *testing.T) {
 			fail: true,
 		},
 
+		// expected failure: config.digest is not hex hashed format
+		{
+			manifest: `
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "size": 1470,
+    "digest": "sha256:c86f7763873b6c0aae22d9h3bab59b4f5debbed6685761b5951584f6efb0633b"
```

Comment:
fixed in https://github.com/opencontainers/image-spec/pull/444/commits/003ac5f168dbc83f3793352ce4820975c8173c08#diff-6493def706b32045f8d30e7a4b860c46R101